### ISSUE TITLE
.osu File gets deleted after saving or creating it at random moment

### ISSUE THREAD

**Issue by Dopaminos** [ppy/osu-stable-issues#1071]
I was mapping a new difficulty for my mapset, but after trying to save it, i got an error "Unable to write file (etc.)".
Because of this, difficulty that i was mapping got deleted. Important detail: It happens only if your audio is .ogg
Restarting the game may help in most of the cases, but this bug happens randomly

Video of bug reproducing and map .osz file is attached.

https://user-images.githubusercontent.com/42616718/207844177-d2e6cbcc-3d61-4ff6-9cb5-1a82cd600ad6.mp4

[UNDER17 - Oyogenai (Cut Ver ) rename it to .osz](https://user-images.githubusercontent.com/42616718/207843856-b863854c-cad5-4f95-8fff-a99b22bacdb6.gif)

upd: nevermind it happens in maps with .mp3 too 
![image](https://user-images.githubusercontent.com/42616718/207901837-8c2644c8-b891-4ab7-a2d3-49ce620ec02b.png)

**Comment by BK8000L** [comment]
lol why my issue (horrible frametimes because of... https://github.com/ppy/osu-stable-issues/issues/1070) has been removed!?

**Comment by smoogipoo** [comment]
- Do you have the file open in another application, perhaps a text editor or maybe another tool like `StreamCompanion`. Try closing every other application related to osu!.
- Is your drive almost full?
- Are you able to change the file manually, e.g. via a text editor, and if doing so does it prompt you for administrator privileges?

Those are some things I can think of off the top of my head.

**Comment by Dopaminos** [comment]
> Do you have the file open in another application, perhaps a text editor or maybe another tool like StreamCompanion. Try closing every other application related to osu!.

No. Here is the my opened tools, no one of them is related to osu
![image](https://user-images.githubusercontent.com/42616718/208041388-6205e511-8681-4213-820a-5ecd56c6b3df.png)

> Is your drive almost full?

No? I guess... (ГБ is GB)
![image](https://user-images.githubusercontent.com/42616718/208041594-2855ab85-b89c-496b-ac97-c88988869dcc.png)

> Are you able to change the file manually, e.g. via a text editor, and if doing so does it prompt you for administrator privileges?

It DELETES .osu file and i can't interact with it, but i can change any other .osu files
